### DISCUSSION TITLE
Aufzeichnung einer Tour teilen

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#486] [Q&A] [answered]
Wie kann ich eine aufgezeichnete Tour teilen so das sie 1:1 beim Empfänger ankommt ?
Als Overlay kann ich sie nicht versenden.
Wenn ich sie als Overlay + Route versenden will stimmt sie nicht genau.
Wenn ich das Overlay mit Wegpunkten ergänzen will um die Route ans Overlay anzupassen kommt eine " verrückte " Route heraus.
Das bedeutet ich kann eine gefahrene und aufgezeichnete Tour nicht teilen ? 🤔
![Screenshot_20240409_172958_Cruiser](https://github.com/devemux86/cruiser/assets/124886589/38802a38-c411-4240-97f1-e48789c11837)
![Screenshot_20240409_173039_Cruiser](https://github.com/devemux86/cruiser/assets/124886589/6ebf7ed5-4fe8-463c-8a9e-749c9f875e65)

Wenn ich das Overlay mit zusätzlichen Wegpunkten ergänzen will kommt eine teilweise verrückte Route heraus.
![Screenshot_20240409_173328_Cruiser](https://github.com/devemux86/cruiser/assets/124886589/256d473e-ca30-4413-8efc-a1df996f7dca)

**Comment by devemux86** [comment]
@1maxi2 

You can share (or export) the route file, it will be exactly as you designed it. Use the menu "Routing | Export".
Recorded tracks are not routes. To make them routes you need to add waypoints and recalculate the route.
Then it depends on the selected routing profile and options to have the same result.

Sie können die Routendatei freigeben (oder exportieren), sie wird genau so sein, wie Sie sie entworfen haben. Verwenden Sie das Menü "Routing | Export".
Aufgezeichnete Tracks sind keine Routen. Um sie zu Routen zu machen, müssen Sie Wegpunkte hinzufügen und die Route neu berechnen.
Dann hängt es vom gewählten Routingprofil und den Optionen ab, ob das gleiche Ergebnis erzielt wird.

**Comment by devemux86** [comment, marked as answer]
@1maxi2 

If you want to share the recorded track, use the menu "GPS recording | Export".

Wenn Sie die aufgezeichnete Spur weitergeben möchten, verwenden Sie das Menü "GPS-Aufzeichnung | Export".

**Reply by 1maxi2** [reply]
O.k. Danke Da ist etwas bei mir schiefgelaufen.
Werde das nach der nächsten Tour noch mal versuchen.
Ich möchte auf alle Fälle den Track teilen damit sichergestellt ist damit er 1:1 beim Empfänger ankommt.
Kann man eigentlich im Menü " GPS Aufzeichnung" einen Button " Teilen" hinzufügen ?

**Reply by devemux86** [reply]
> Ich möchte auf alle Fälle den Track teilen damit sichergestellt ist damit er 1:1 beim Empfänger ankommt.

You can export recorded tracks and share them with other users.
But the other users will receive and import tracks (not routes).

Sie können aufgezeichnete Tracks exportieren und sie mit anderen Benutzern teilen.
Aber die anderen Benutzer erhalten und importieren Tracks (nicht Routen).

> Kann man eigentlich im Menü " GPS Aufzeichnung" einen Button " Teilen" hinzufügen ?

It is available in the Cruiser Beta:

Sie ist bereits in der Cruiser-Beta verfügbar:
- https://github.com/devemux86/cruiser/discussions/482

**Reply by 1maxi2** [reply]
Das ist so gewünscht das andere Benutzer den Track erhalten. So können Sie anhand des Track selber die Route planen so das sie übereinstimmt

**Reply by DIrgi** [reply]
@1maxi2 

Wenn Dir Dein Moped lieb ist und Du etwas Kleingeld zur Verfügung hast (so einmalig 250€), kann ich Dir einen GPS Tracker empfehlen. Der dient zwar primär dazu Dein Motorrad zu Tracken, also als Diebstalsicherung und Bewegungsmelder etc., aber neben bei macht er noch "kostenlos" Echtzeit Tracking zum Teilen, Wenn Dich das Interessiert, melde Dich.

**Reply by 1maxi2** [reply]
Nein Danke. Möchte nur wenn eine ungeplante Tour besonders schön war sie anderen Benutzer teilen. Dafür reicht die Aufzeichnung mit dem Smartphone
